src/app/doc-pages/paginator-doc/paginator-doc.component.html

```diff
@@ -0,0 +1,55 @@
+<div class="doc-page-container">
+
+  <h1 class="doc-title">Paginator Component</h1>
+
+  <p class="doc-info">
+    Pagination provides navigation to the page, by sharing content across several separate pages
+    and linking them to the bottom / top of the page. 
+    This way you give the user a sense of control by knowing where they are and they can estimate how much time it will take to find it.
```

Comment:
Vurder noe ala:
Pagination enables tables content (client or serverside) to be dividet up into smaller pages that can be navigated using this paginator.